test_runner/regress/test_compute_catalog.py

```diff
@@ -370,3 +372,66 @@ def test_compute_drop_role(neon_simple_env: NeonEnv):
         cursor.execute("SELECT rolname FROM pg_roles WHERE rolname = 'readonly2'")
         role = cursor.fetchone()
         assert role is None
+
+
+def test_drop_role_with_table_privileges_from_non_neon_superuser(neon_simple_env: NeonEnv):
+    """
+    Test that compute_ctl can drop a role if the role has previously been
+    granted table privileges by a role other than neon_superuser.
+    """
+    TEST_DB_NAME = "neondb"
+    TEST_GRANTOR = "my_grantor"
+    TEST_GRANTEE = "my_grantee"
```

Comment:
Regardless of whether the current code is prone to injections or not (I bet it does, though), let's make role names for both grantor and grantee to contain the injections vector like `'; RAISE 'ouch'; '` or `"; RAISE 'ouch'; "` depending on the type of quotes you use in what case

Reply:
Good idea.